### PR TITLE
fixes singular vs. plural text + speak of "bids" instead of "buys" in case of an auction

### PR DESCRIPTION
**Pull request by dontbuyrope**

### CODE REVIEW

File: src/components/Lists/ListOfBuyers.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="list-of-buyers rounded-container" :class="darkMode ? 'dark-mode-surface no-border' : 'light-mode-surface'">
     <template v-if="list.length === 0">
       <div class="text-center text-gray-400">
-        No buyers yet! Be the first!
+        {{ isAuction ? "No bids yet! Be the first!" : "No buyers yet! Be the first!" }}
```
Comment (dontbuyrope) [inline src/components/Lists/ListOfBuyers.vue:5]: more in line with the title of this block: Recent bids vs. Recent buys

**Review by JayWelsh — APPROVED** [review]
Thanks! 💯

**Comment by JayWelsh** [comment]
Looks good, thanks @dontbuyrope! Feel free to merge when you are ready and the change will be auto deployed. 🌈

**Comment by dontbuyrope** [comment]
would love to, but don't have write access to this repository tho

**Comment by JayWelsh** [comment]
Merged, thanks a lot for your help @dontbuyrope ! Really appreciate it!

**Pull request merged**